Add basic date display on right of package list

michaeljones opened the pull request:
Not sure this is the best location but it saves on the hassle of trying to get them to neatly line up with the headers which is tricky due to the different font sizes and maybe the underline.

Currently displays an ugly iso8601 format for the date. Would be better as '3 days ago' style display but I'm not sure of the best package/lib to help with that.

![image](https://user-images.githubusercontent.com/5390/235366504-5a3b1c80-9db6-4abf-8a92-d3181d8b9aa9.png)

michaeljones commented:
I think a `YYYY-MM-DD` format might be a reasonable fallback if a '3 days ago' approach is harder to come by. `birl` doesn't seem to offer easy access to a `YYYY-MM-DD` format though we could just split the current output on the `T`. Otherwise it is a bit more work to go via the `Month` type and other things but obviously not hard.

lpil commented on priv/styles.css:
```diff
@@ -120,10 +121,24 @@ h1, h2, h3, h4, h5, h6 {
 }
 
-.package-list h2 {
+.package-list li div {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-end;
+  gap: var(--gap);
+
   margin: 0 var(--gap) var(--gap-s) 0;
 }
 
+.package-list li div span {
```
Let's add classes to these elements rather than using the type of the element. Make it a bit clearer and avoid accidentally applying this to other in future.

michaeljones replied:
Good point. Done!

lpil reviewed (commented):
Thank! A shorter date format would be good.

Might be easier to position the date with `float: right`?

michaeljones commented:
All makes sense. I'll try to have a look over the next day or two. No worries if someone comes along with a bigger change that conflicts though.

michaeljones commented:
Updated as you requested (I forgot about 'float'!)

And updated to birl 0.11 to use 'legible_difference'.

![image](https://user-images.githubusercontent.com/5390/236631727-2626b55f-ad7c-483a-af26-4c92f1ec9d4f.png)

lpil reviewed (approved):
Thanks!!
